deploy-collector.input.sample.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
I wouldn't remove this sample file, because it may simplify the usage.

Reply:
this was removed by mistake